sdks/java/fn-execution/src/main/java/org/apache/beam/sdk/fn/data/BeamFnDataGrpcMultiplexer.java

```diff
@@ -48,28 +44,23 @@
  */
 public class BeamFnDataGrpcMultiplexer {
   private static final Logger LOG = LoggerFactory.getLogger(BeamFnDataGrpcMultiplexer.class);
-  private final Endpoints.ApiServiceDescriptor apiServiceDescriptor;
```

Comment:
What was the reasoning behind removing this, its very useful for logging.

Reply:
Ping?

Reply:
I've made it `@Nullable` instead of removing it - it's not always available (e.g. in `GrpcDataService` as I've written it it's not available at the time the multiplexer is constructed)

Reply:
If its not available then its not available. I was just wondering why it didn't exist, your choice to keep it or remove it but it looks cleaner without it though now that it can be nullable.

Reply:
I'm not bothered by it being available only sometimes if it's useful